docs/tutorial/fdfitting.rst

```diff
@@ -174,6 +174,14 @@ when loading the data from `DNA/Lc` to `DNA/Lc_RecA`::
     distance2 = fd2.d[mask2].data
     fit.add_data("RecA", force2, distance2, params={"DNA/Lc": "DNA/Lc_RecA"})
 
+Multiple criteria can be combined into a single mask by using `numpy`'s logical operators.
+For example, if we want to select all forces up to `30` pN, and only include distances between `1.5` and `4.0` micron, we could create the mask by invoking::
+
+    example_mask = np.logical_and(np.logical_and(fd1.f.data <= 30, fd1.d.data > 1.5), fd1.d.data < 4.0)
+
+Or more succinctly (note the double brackets)::
+
+    example_mask = np.logical_and.reduce((fd1.f.data <= 30, fd1.d.data > 1.5, fd1.d.data < 4.0))
```

Comment:
Multiple criteria can be combined into a single mask by using `numpy`'s logical operators.
For example, if we want to select all forces up to `30` pN and distance data larger than '1.5' micron: 

    example_mask = np.logical_and(d1.f.data <= 30, fd1.d.data > 1.5)

Adding a third condition that the distance has to be < 4 micron (note the double brackets)::

    example_mask = np.logical_and.reduce((fd1.f.data <= 30, fd1.d.data > 1.5, fd1.d.data < 4.0))

Reply:
Good point! Amending!